lib/generators/solidus_graphql_api/install/install_generator.rb

```diff
@@ -22,6 +25,18 @@ def run_migrations
           puts 'Skipping bin/rails db:migrate, don\'t forget to run it!' # rubocop:disable Rails/Output
         end
       end
+
+      def install_routes
+        if Pathname(app_path).join('config', 'routes.rb').read.include? MOUNT_GRAPHQL_ENGINE
+          say_status :route_exist, MOUNT_GRAPHQL_ENGINE, :blue
+        else
+          route <<~RUBY
+            # This line installs graphql's main route to execute queries
+
+            mount SolidusGraphqlApi::Engine, at: '/'
```

Comment:
That will override whatever was mounted in the root namespace, typically the regular HTML frontend. I think we can mount into a "/graphql" namespace and define a `POST /` route there. However, that will be a breaking change, as the new GraphQL endpoint will be "/graphql/"  instead of "/graphql" (with slash at the end). I think we can do that as we're still in a `0.x` phase.

Reply:
Ah you are right whoops! That could be an issue :laughing: .

I've updated the code to do as you suggest and tried it, it works fine with my Altair GraphQL client and queries so I think we're good. Also in `bin/rails routes` I get:
```
zach@zach-lotus:~/dev/marketplace (main)$ bin/rails routes
                                         Prefix Verb     URI Pattern                                                                                            Controller#Action
                            solidus_graphql_api          /graphql                                                                                               SolidusGraphqlApi::Engine
    spree_user_google_oauth2_omniauth_authorize GET|POST /users/auth/google_oauth2(.:format)
```

and later on:
```
Routes for SolidusGraphqlApi::Engine:
       POST /           spree/graphql#execute
```

seems to work well.